<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindow</class>
 <widget class="QDialog" name="ConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lvlLabel">
      <property name="text">
       <string>A quelle niveau souhaitez vous commencez le jeu ?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lvlEdit">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lvlErr">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="rowLabel">
      <property name="text">
       <string>Combien de lignes voulez vous ?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="rowEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="rowErr">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="colLabel">
      <property name="text">
       <string>Combien de colonnes voulez vous ?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="colEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="colErr">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="fillGrid">
      <property name="text">
       <string>Voulez vous remplir partiellement la grille ?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="submitBtn">
      <property name="text">
       <string>Commencer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="configLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Configuration du jeu :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
